metpy/plots/declarative.py

```diff
@@ -537,14 +541,63 @@ class MapPanel(Panel):
 
     parent = Instance(PanelContainer)
     layout = Tuple(Int(), Int(), Int(), default_value=(1, 1, 1))
+    layout.__doc__ = """A tuple that contains the description (nrows, ncols, index) of the
+    panel position; default value is (1, 1, 1).
+
+    This trait is set to describe the panel position and the default is for a single panel. For
+    example, a four-panel plot will have two rows and two columns with the tuple setting for
+    the upper-left panel as (2, 2, 1), upper-right as (2, 2, 2), lower-left as (2, 2, 3), and
+    lower-right as (2, 2, 4). For more details see the documentation for Matplotlib subplots.
+    """
+
     plots = List(Any())
+    plots.__doc__ = """A list of handles that represent the plots (e.g., ContourPlot,
+    FilledContourPlot, ImagePlot) to put on a given panel.
+
+    This trait collects the different plots, including contours and images, that are intended
+    for a given panel.
+    """
+
     _need_redraw = Bool(default_value=True)
 
     area = Union([Unicode(), Tuple(Float(), Float(), Float(), Float())], allow_none=True,
                  default_value=None)
+    area.__doc__ = """A tuple or string value that indicates the graphical area of the plot.
+
+    The tuple value coresponds to longitude/latitude box based on the projection of the map
+    with the format (west-most longitude, east-most longitude, south-most latitude,
+    north-most latitude). This tuple defines a box from the lower-left to the upper-right
+    corner.
+
+    This trait can also be set with a string value associated with the named geographic regions
+    within MetPy. The tuples associated with the names are based on a PlatteCarree projection.
+    For a CONUS region, the following strings can be used: 'us', 'spcus', 'ncus', and 'afus'.
+    For regional plots, US state postal codes can be used.
+    """
+
     projection = Union([Unicode(), Instance(ccrs.Projection)], default_value='data')
+    projection.__doc__ = """A string for a pre-defined projection or a Cartopy projection
+    object.
+
+    There are three pre-defined projections that can be called with a short name:
+    Lambert Conformal ('lcc'), Mercator ('mer'), or polar-stereographic ('ps'). Additionally,
```

Comment:
Should "Conformal" be "conformal"?

Reply:
I've seen it both ways...we'll go lower case hear and I added "conic" to make it a more inline with using "lcc"

Reply:
I'm ok with leaving it capital if polar stereographic is also capitalized.